pkg/cosign/tuf/client.go

```diff
@@ -251,7 +251,7 @@ func GetTarget(ctx context.Context, name string, out client.Destination) error {
 	return getTargetHelper(name, out, c)
 }
 
-func getRootKeys(rootFileBytes []byte) ([]*data.Key, int, error) {
+func getRootKeys(rootFileBytes []byte) ([]*data.PublicKey, int, error) {
```

Comment:
cc @asraa when I updated the intoto dependency it automatically updated `go-tuf` as well. Just wanted to make sure this looks correct to you?

Reply:
resolved offline!